### ISSUE TITLE
locals of type exnref

### ISSUE THREAD

**Issue by Horcrux7** [WebAssembly/wabt#1388]
If try to compile the follow sample code with the command line: `wat2wasm.exe --enable-all sample.wat`

```
(module
    (func $test(local $exn exnref)
        try
          i32.const 0
		  drop
        catch
		  local.tee $exn
          drop
        end
    )
)
```
then I receive the error:
```
sample.wat:2:28: error: unexpected token "exnref", expected i32, i64, f32, f64, v128 or anyref.
    (func $test(local $exn exnref)
                           ^^^^^^

```

Because try/catch from the exception proposal is supported WABT should also supports the variable type `exnref`.

I have use the master version from today on Windows.

**Comment by binji** [comment]
Whoops, thanks for mentioning! Should be fixed in #1389.

**Comment by Horcrux7** [comment]
Works.
